### PR TITLE
app-admin/logstash-bin: bump to 5.6.9/6.2.4 / drop old

### PR DESCRIPTION
**Pull request by hydrapolic**
Thanks :)

### CODE REVIEW

File: app-admin/logstash-bin/logstash-bin-5.6.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	keepdir /etc/"${MY_PN}"/{conf.d,patterns,plugins}
+	keepdir "/var/lib/${MY_PN}"
+	keepdir "/var/log/${MY_PN}"
+
+	insinto "/usr/share/${MY_PN}"
+	newins "${FILESDIR}/agent.conf.sample" agent.conf
+
+	rm -v config/startup.options || die
+	insinto /etc/${MY_PN}
+	doins config/*
```
Comment (Zlogene) [inline app-admin/logstash-bin/logstash-bin-5.6.9.ebuild]: please avoid globs and use `doins -r` here

File: app-admin/logstash-bin/logstash-bin-5.6.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+EAPI=6
+
+inherit java-pkg-2 user
+
+MY_PN="${PN%-bin}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Tool for managing events and logs"
+HOMEPAGE="https://www.elastic.co/products/logstash"
+SRC_URI="https://artifacts.elastic.co/downloads/${MY_PN}/${MY_P}.zip"
```
Comment (Zlogene) [inline app-admin/logstash-bin/logstash-bin-5.6.9.ebuild]: `app-arch/unzip` to DEPEND (in older ebuilds better too)

File: app-admin/logstash-bin/logstash-bin-5.6.9.ebuild [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+	ewarn "Please ensure any running logstash processes are shut down cleanly."
+	ewarn
+	ewarn "The default data directory has been moved from /opt/logstash/data to"
+	ewarn "/var/lib/logstash/data. Please check and move its contents as necessary."
+	ewarn
+	ewarn "Self installed plugins are removed during Logstash upgrades (Bug #622602)"
+	ewarn "Install the plugins via eselect module that will automatically re-install"
+	ewarn "all self installed plugins after Logstash upgrades."
+	einfo
+	einfo "Installing plugins:"
```
Comment (mgorny) [inline app-admin/logstash-bin/logstash-bin-5.6.9.ebuild]: Why are those messages `einfo`? `einfo` is only for progress messages that are irrelevant once build is complete. If those are instructions for the user, use `elog` instead.

Comment (hydrapolic) [reply app-admin/logstash-bin/logstash-bin-5.6.9.ebuild]: Actually haven't noticed the difference, good point, thanks.

**Comment by hydrapolic** [comment]
Thanks @mgorny, pushed an update.

**Comment by mgorny** [comment]
Thanks!

**Comment by hydrapolic** [comment]
Thank you @mgorny

**Pull request closed without merging**